[IMP] l10n-it: modules and e-invoicing updates (#16134)
odoo/documentation: content/applications/finance/fiscal_localizations/italy.rst

   * - Italy - E-invoicing
     - `l10n_it_edi`
     - E-invoice implementation
   * - Italy - Declaration of Intent
     - `l10n_it_edi_doi`
     - Support for the Declaration of Intent (Dichiarazione di Intento)

>    * - Italy - Italy - E-invoicing - Additional module to support the debit notes (nota di debito -

Reviewer: ```suggestion
   * - Italy - E-invoicing - Additional module to support the debit notes (nota di debito -
```

       NDD)
     - `l10n_it_edi_ndd`

>      - Support for debit notes (nota di debito - NDD) by adding payment methods and document types

Reviewer: ```suggestion
     - Adds  payment methods and document types for debit notes
```
(and I don't think we need the Italian translation here since it's a "normal" translation and this will be translated)

   * - Italy - Italy - E-invoicing - Bridge module between Italy NDD and Account Debit Note
     - `l10n_it_edi_ndd_account_dn`
     - Support for debit notes (nota di debito - NDD) by adding debit note fields
   * - Italy - Sale E-invoicing
     - `l10n_it_edi_sale`
     - Sale modifications for E-invoicing
   * - Italy eCommerce eInvoicing
     - `l10n_it_edi_website_sale`
     - Italian eCommerce eInvoicing features
   * - Italy - E-invoicing (Withholding)
     - `l10n_it_edi_withholding`
     - E-invoice withholding
   * - Italy - Point of Sale
     - `l10n_it_pos`
     - Integration of Odoo PoS with the Italian Fiscal Printer
   * - Italy - Accounting Reports
     - `l10n_it_reports`

>      - Country-specific reports

Reviewer: ```suggestion
     - Italian reports
```
